data/language/en-GB.txt

```diff
@@ -2953,7 +2953,7 @@ STR_5858    :Use GPU for displaying instead of CPU. Improves compatibility with
 STR_5859    :Enables frame tweening for visually{NEWLINE}smoother gameplay. When disabled,{NEWLINE}the game will run at 40 FPS.
 STR_5860    :Toggle original/decompiled track drawing
 STR_5861    :Key verification failure.
-STR_5862    :Block unknown players.
+STR_5862    :Block unknown players
 STR_5863    :Only allow players with known keys to join.
```

Comment:
Maybe we should remove the trailing period here as well. No checkbox label should use a period, as far as I'm concerned.